webpack.config.js

```diff
@@ -77,13 +76,10 @@ module.exports = {
     new BrowserSyncPlugin(
       // BrowserSync options
       {
-        // browse to http://localhost:3000/ during development
-        host: 'localhost',
-        port: 3000,
         // proxy the Webpack Dev Server endpoint
         // (which should be serving on http://localhost:3100/)
         // through BrowserSync
-        proxy: 'http://127.0.0.1:5000',
+        proxy: 'https://littlespain.localtest.me:8000/',
```

Comment:
Should this be specific for this single account?

Reply:
this is the only site in the database that currently uses this theme, so no other URL will show your changes to the theme